packages/devtools_app/lib/src/screens/memory/framework/offline_data/offline_data.dart

```diff
@@ -21,10 +21,11 @@ class OfflineMemoryData {
     this.profile,
     this.chart,
     this.filter, {
-    this.isEmpty = false,
     required this.selectedTab,
   });
 
+  // TODO: use an extension type for the Json parsing, https://github.com/flutter/devtools/issues/6972
```

Comment:
is this the right issue?

Reply:
Yes, I want to have it done before completing the disconnect experience. Most likely next PR.